### DISCUSSION TITLE
微信目录下 config.ini问题

### DISCUSSION THREAD

**Discussion by liubowei2018** [ttttupup/wxhelper#239] [Q&A]
单个微信一一启动，会根据config.ini中的port进行，但是已经启动比如两个微信 第一个注入就是默认的19088，这是什么问题了

**Comment by ttttupup** [comment]
GetPrivateProfileInt这个函数的一些问题避免了就不会出现这种情况。

**Reply by liubowei2018** [reply]
有什么需要注意的吗

**Reply by ttttupup** [reply]
msdn查一下
